Cargo.toml

```diff
@@ -57,43 +57,42 @@ futures-io = "0.3.14"
 futures-util = { version = "0.3.14", features = ["io"] }
 futures-executor = "0.3.14"
 hex = "0.4.0"
-hmac = "0.11"
+hmac = "0.12.1"
 lazy_static = "1.4.0"
-md-5 = "0.9.1"
+md-5 = "0.10.1"
 openssl = { version = "0.10.38", optional = true }
 openssl-probe = { version = "0.1.5", optional = true }
 os_info = { version = "3.0.1", default-features = false }
 percent-encoding = "2.0.0"
 rand = { version = "0.8.3", features = ["small_rng"] }
-rustls-pemfile = "0.2.1"
+rustls-pemfile = "0.3.0"
 serde_with = "1.3.1"
-sha-1 = "0.9.4"
-sha2 = "0.9.3"
+sha-1 = "0.10.0"
+sha2 = "0.10.2"
 snap = { version = "1.0.5", optional = true}
 socket2 = "0.4.0"
 stringprep = "0.1.2"
 strsim = "0.10.0"
 take_mut = "0.2.2"
 thiserror = "1.0.24"
 tokio-openssl = { version = "0.6.3", optional = true }
-trust-dns-proto = "0.20.0"
-trust-dns-resolver = "0.20.0"
-typed-builder = "0.9.0"
+trust-dns-proto = "0.21.1"
```

Comment:
I ended up needing to bump these rather than waiting for #591 because the older versions were incompatible with the new version of `async-std-resolver`.